handbook/recruitment/designers.md

```diff
@@ -0,0 +1,52 @@
+---
+# this page is meant to be sent to non-Launch Pad members as recruitment material - exclude it from search
+search: false
+---
+
+# Designers 🚀 <img align="right" src="https://raw.githubusercontent.com/ubclaunchpad/ubclaunchpad.com/master/src/assets/rocket.png" width="100px">
```

Comment:
```suggestion
# Designer 🚀 <img align="right" src="https://raw.githubusercontent.com/ubclaunchpad/ubclaunchpad.com/master/src/assets/rocket.png" width="100px">
```

(for consistency with singular [tech lead](https://docs.ubclaunchpad.com/handbook/recruitment/tech-lead))

could you change the name of this file as well? :)